modules/sdk-coin-avaxc/test/unit/avaxc.ts

```diff
@@ -619,4 +619,54 @@ describe('Avalanche C-Chain', function () {
         .should.be.rejectedWith('coin in txPrebuild did not match that in txParams supplied by client');
     });
   });
+
+  // TODO move to modules/bitgo/test/v2/integration/coins/avaxc.ts
```

Comment:
created ticket for you: https://bitgoinc.atlassian.net/browse/BG-56136

can reference here